utils.py

```diff
@@ -273,7 +273,7 @@ def verifyTestLog (report, package, mustpass, fractionMustpass, gitSHA):
 				anyErrorTmp, resultOrderOk = validateTestCasePresence(report, fractionMustpass, results)
 				anyError |= anyErrorTmp
 
-				if anyError:
+				if anyErrorTmp:
```

Comment:
why have you changed this?

Reply:
`anyError` accumulates all previous errors, leading to spurious printouts if any other (unrelated) error is emitted before.